### ISSUE TITLE
Suggestion: Add "constellation-finder" based on GPS and phone rotation

### ISSUE THREAD

**Issue by vanitasvitae** [gituser1357/constellations#1]
Hi
It would be a really really nice feature, if constellations could show you where a certain constellation is, based on your GPS coordinates and the rotation of the phone. That way you could aim for the sky and see (maybe as a virtual sky sphere) the constellations highlighted.

I know that would be a complex feature, but it would be great :)
Also I hope you understand what I mean, English is not my first language.

**Comment by gituser1357** [comment]
Yes, it would be good to have that interface, but it requires 3d/opengl programming and also collection/integration of astronomical data to be as detailed as the existing IAU maps. Possibly at some point in the future I will take this up but it will be a big effort.
